### PR TITLE
Commit for v700

### PR DESCRIPTION
**Pull request by SwagatoMondal**
Commit for v700

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ To download the latest SDK as a JAR, please visit [http://inmobi.com/sdk](https:
 **To continue integrating with the InMobi SDK, please see the [Integration Guidelines](https://support.inmobi.com/monetize/android-guidelines/) for Android.**
 
 ## New in this version
-InMobi SDK v6.2.4 for Android introduces the ability to monetize the lock screen.
+InMobi SDK v7.0.0 for Android introduces the ability to monetize the lock screen.
```
Comment (amit-prabhudesai) [inline README.md]: The description here needs update.

File: samples/nativeSample/build.gradle [hunk 2]
```diff
@@ -5,47 +5,40 @@ android {
...
     compile "com.android.support:cardview-v7:${androidSupportLibVersion}"
-
     compile "com.google.android.gms:play-services:${gmsVersion}"
-    compile "com.facebook.fresco:fresco:0.5.0"
-    compile "com.squareup.picasso:picasso:2.5.2"
+
+    compile 'com.facebook.fresco:fresco:0.5.0'
+    compile 'com.squareup.picasso:picasso:2.5.2'
     compile 'com.android.support:multidex:1.0.1'
+    compile 'com.android.support.constraint:constraint-layout:1.0.2'
```
Comment (amit-prabhudesai) [inline samples/nativeSample/build.gradle]: Are we actually using `ConstraintLayout`s?

File: samples/nativeSample/src/main/java/com/inmobi/nativead/PlacementId.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package com.inmobi.nativead;
+
+/**
+ * Placement ID to be used to render the Native ads.
+ */
+public interface PlacementId {
+   long YOUR_PLACEMENT_ID_HERE = 1492763427180L;
+}
```
Comment (amit-prabhudesai) [inline samples/nativeSample/src/main/java/com/inmobi/nativead/PlacementId.java]: Nit: No newline at end-of-file.

File: samples/nativeSample/src/main/java/com/inmobi/nativead/sample/singlestrand/SingleStrandFragment.java [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+import com.inmobi.nativead.sample.R;
+import com.inmobi.nativead.utility.SwipeRefreshLayoutWrapper;
+import com.squareup.picasso.Picasso;
+
+/**
+ * Demonstrates the use of InMobiNativeStrand to place a single ad.
+ * <p/>
+ * Note: Swipe to refresh ads.
+ */
+public class SingleStrandFragment extends Fragment
```
Comment (amit-prabhudesai) [inline samples/nativeSample/src/main/java/com/inmobi/nativead/sample/singlestrand/SingleStrandFragment.java]: We no longer have "strands", so the class name and the class-level Javadoc needs to  change.

**Review by amit-prabhudesai — CHANGES_REQUESTED** [review]
Please check the notes. Also, I suppose the recommended Google Play Services version should still be 8.4.0? This was changed in the 6.2.4 update, but should be set to the version that QA has certified with.

**Comment by SwagatoMondal** [comment]
@amit-prabhudesai Added changes as per review comments. Please review.

**Review by amit-prabhudesai — APPROVED** [review]
Looks good.

**Pull request merged**